src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -59,11 +72,11 @@
 }
 
 <script asp-name="removeEmptyActionsMenu" at="Foot" type="text/javascript">
-    $(function(){
-        
-        $(".actions-menu.dropdown-menu").each(function(i, e){
+    $(function () {
+
+        $(".actions-menu.dropdown-menu").each(function (i, e) {
```

Comment:
Half these checks are now done in the driver? 
And the other half are still done in the views.

You said that was to not have an empty button when there were no items in the list?

So do we need the script / not need the script as well?

Reply:
That might be an optimization but not necessary.

Reply:
Ok I removed this code entirely as now it's shown/hidden appropriately based on permissions the user has.